workflow/sample_files/base-enclosure-2stories-garage.xml

```diff
@@ -207,7 +207,7 @@
             <ExteriorAdjacentTo>ground</ExteriorAdjacentTo>
             <InteriorAdjacentTo>basement - conditioned</InteriorAdjacentTo>
             <Height>8.0</Height>
-            <Area>880.0</Area>
+            <Area>1200.0</Area>

```

Comment:
Good change, this does look right. Seems like it was using the _exposed_ perimeter of the basement to calculate the area (110x8=880) instead of the _total_ perimeter (150x8=1200).